src/widgets/ReportTable/ReportTable.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2018
+ */
+
+import React from 'react';
+import { StyleSheet, FlatList, View } from 'react-native';
+import PropTypes from 'prop-types';
+import { LIGHT_GREY } from '../../globalStyles';
+import { ReportRow } from './ReportRow';
+
+export const ReportTable = ({ rows, header }) => {
+  const renderItem = ({ item, index }) => <ReportRow rowData={item} rowIndex={index} />;
+
+  const renderHeader = () => <ReportRow rowData={header} isHeader rowIndex={0} />;
```

Comment:
This is fine, but there is also this prop on `FlatList` just for interest!  https://facebook.github.io/react-native/docs/flatlist#listheadercomponent

Reply:
I made this change here: 7218d46
Hope it works, otherwise I can come back to previous stage :-) thanks for this!

Reply:
I think that should work!